apps/allocations/app/components/Panel/NewBudget.js

```diff
@@ -41,12 +41,13 @@ class NewBudget extends React.Component {
     super(props)
     this.state =  INITIAL_STATE
     if (props.editingBudget.id) {
+      const tokenIndex = props.tokens.findIndex(token => token.symbol === props.editingBudget.token.symbol)
       this.state.name = props.editingBudget.name
       this.state.nameError = false
       this.state.amount = BigNumber(props.editingBudget.amount)
         .div(ETH_DECIMALS)
       this.state.amountError = false
-      this.state.selectedToken = props.editingBudget.token.symbol === 'ETH' ? 0 : 1
+      this.state.selectedToken = tokenIndex === -1 ? 0 : tokenIndex //Default to ETH if index not found?
```

Comment:
Why the question mark?

Reply:
Oops! I wrote that with the plan to follow up on what we wanted for the default behaviour. So if we cannot find the token symbol among the props.tokens, what behaviour do we want? Should we just default to ETH or should we have some error message?

Reply:
Seems like an errors state to me!

Reply:
I decided to take another route. Currently, for editing the budget we have a bit of a roundabout way of displaying the symbol, e.g. `symbols[selectedToken]`. But we already know the token symbol -- `props.editingBudget.token.symbol` so I decided to just display that value in the panel and set `this.state.selectedToken = 0` since we don't need to know the token to update the budget. This way we don't need to do any error handling.